src/components/fields/checkbox-group/checkbox-group.styles.tsx

```diff
@@ -1,10 +1,16 @@
 import { Checkbox } from "@lifesg/react-design-system/checkbox";
 import { Text } from "@lifesg/react-design-system/text";
 import styled from "styled-components";
+import { TCheckboxToggleLayoutType } from "./types";
+
 interface ILabelProps {
 	disabled?: boolean | undefined;
 }
 
+interface IToggleWrapperProps {
+	layoutType: TCheckboxToggleLayoutType;
```

Comment:
prepend transient prop with `$`

```suggestion
	$layoutType: TCheckboxToggleLayoutType;
```